src/Http/Controllers/Auth/ForgotPasswordController.php

```diff
@@ -8,6 +8,7 @@
 use GetCandy\Api\Http\Resources\Auth\PasswordTokenResource;
 use Illuminate\Foundation\Auth\SendsPasswordResetEmails;
 use Illuminate\Support\Facades\Password;
+use GetCandy\Api\Core\Users\Actions\FetchUser;
 
 class ForgotPasswordController extends BaseController
```

Comment:
Same here, do we need this controller?

Reply:
These were just moved over as to not break them currently but yes there are a couple like this that would just move to an action, I can do that as part of this if that's easier?

Reply:
I think since we've started password reset actions it would make sense to convert them across if they are related.

Reply:
That's fine in that respect, I was thinking really FetchImpersonationToken and the reset passwords should probably live in Auth in that case too?